dotnetv3/Glue/README.md

```diff
@@ -0,0 +1,73 @@
+# AWS Glue code examples for the AWS SDK for .NET (v3)
+
+## Overview
+
+The examples in this folder show how to use AWS Glue with the AWS SDK for .NET.
+
+AWS Glue is a serverless data integration service that makes it easy to 
+discover, prepare, and combine data for analytics, machine learning, and
+application development. AWS Glue provides all the capabilities needed for data
+integration so that you can start analyzing your data and putting it to use in
+minutes instead of months.
+
+## ⚠️ Important
+
+- Running this code might result in charges to your AWS account.
+- Running the tests might result in charges to your AWS account.
+- We recommend that you grant your code least privilege. At most, grant only
+  the minimum permissions required to perform the task. For more information,
+  see [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+- This code is not tested in every AWS Region. For more information, see [AWS
+  Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/).
+
+## Code examples
+
```

Comment:
Missing Single action example list. Check the [README template](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template#single-actions).

Reply:
I don't have single action examples.